test/verify/check-login

```diff
@@ -561,5 +561,135 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
     def testFailingWebsocketSafariNoCA(self):
         self.testFailingWebsocket(safari=True, cacert=False)
 
+    def testTally(self):
+        m = self.machine
+        b = self.browser
+
+        if m.image.startswith('debian') or m.image.startswith('ubuntu'):
+            module = 'pam_tally2'
+            logfile = lambda user: '/var/log/tallylog'
+        elif m.image.startswith('rhel') or m.image.startswith('fedora') or m.image.startswith('cent'):
+            module = 'faillock'
+            logfile = lambda user: '/var/run/faillock/' + user
+        else:
+            raise ValueError('unexpected image name', m.image)
+
+        def expect_fail_count(expected, user="admin", must_exist=True):
+            if must_exist:
+                cksum = m.execute("cksum {}".format(logfile(user)))
+
+            output = m.execute("{} --user {}".format(module, user))
```

Comment:
How convenient that they both have the same CLI :)

Reply:
It's almost like one started off as a fork of the other ;)